Resources/Prototypes/Entities/Objects/Misc/parcel_wrap.yml

```diff
@@ -35,26 +37,30 @@
   - type: ContainerContainer
     containers:
       contents: !type:ContainerSlot
-  - type: Appearance
-  - type: GenericVisualizer
-    visuals:
-      enum.WrappedParcelVisuals.Size:
-        enum.WrappedParcelVisuals.Layer:
-          "Tiny": { state: "parcel-tiny" }
-          "Small": { state: "parcel-small" }
-          "Medium": { state: "parcel-medium" }
-          "Large": { state: "parcel-medium" }
-          "Huge": { state: "parcel-large" }
-          "Ginormous": { state: "parcel-large" }
+#  - type: Appearance
+#  - type: GenericVisualizer
+#    visuals:
+#      enum.WrappedParcelVisuals.Size:
+#        enum.WrappedParcelVisuals.Layer:
+#          "Tiny": { state: "parcel-tiny" }
+#          "Small": { state: "parcel-small" }
+#          "Medium": { state: "parcel-medium" }
+#          "Large": { state: "parcel-medium" }
+#          "Huge": { state: "parcel-large" }
+#          "Ginormous": { state: "parcel-large" }
```

Comment:
There's a bunch of missing stuff regarding the parcel changing its size based on its contents, and changing its sprite based on its size.